### PR TITLE
test: add test for uploading a single file

### PR DESCRIPTION
**Pull request by alanshaw**
* Adds a mock cluster server using [smoke](https://npm.im/smoke)
* Adds a test that uploads a file, ensures the response is correct and checks it was added to the NFTS and PINS stores
* Minor changes to the JWT utility module to allow it to be used in tests

### CODE REVIEW

File: packages/api/test/mocks/cluster/post_add.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// @ts-ignore
```
Comment (vasco-santos) [inline packages/api/test/mocks/cluster/post_add.js:1]: nit: usually nice to add the reason of the ignore next to this

**Review by vasco-santos — APPROVED** [review]
This looks good!
Left a nit comment and what about `packages/cron/src/bin/pinata.js`? It seems empty, should we delete the file?

**Review by hugomrdias — APPROVED** [review]
LGTM

**Comment by alanshaw** [comment]
> Left a nit comment and what about `packages/cron/src/bin/pinata.js`?

Not empty? https://github.com/ipfs-shipyard/nft.storage/blob/main/packages/cron/src/bin/pinata.js

**Pull request merged**